.github/workflows/publish.yml

```diff
@@ -16,7 +16,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install --upgrade setuptools wheel twine
+        python -m pip install --upgrade setuptools
+        python -m pip install wheel twine
```

Comment:
I'm wondering if we should move this (and the commands below) into a tox environment as well, for consistency? (It would also allow manually uploading a release more easily, although we really shouldn't have to do that anymore now that we have GitHub Actions set up to do that...)

Reply:
Hrm... I completely agree that we *could*; my only hesitation would be making it easy to do something we shouldn't be doing. However, I agree there's value in having all config in one place, so I'll add this in.

Reply:
All this radical simplification does make me wonder if there is (and if not, why there isn't) a Github Action to do the "update pip, update setuptools, install tox, run tox" dance...